index.html

```diff
@@ -2,10 +2,12 @@
 <html>
```

Comment:
Yes.  I didn't have time to load things into the body on init (just after setting the api), but I want to get it there.